FusionIIIT/applications/academic_procedures/views.py

```diff
@@ -408,14 +408,18 @@ def academic_procedures_student(request):
         
 
         # Branch Change Form save
+        # The form will no longer save here. There is different function named as branch_change_request that will handle branch change request
```

Comment:
Don't touch this function, for now. 
I totally understand that this function has no use, but we will handle that in a different issue.

Reply:
noted

Reply:
Can you revert the changes? We will handle this in a different issue.